### DISCUSSION TITLE
Startups ,Businesses , Enterpreneurships

### DISCUSSION THREAD

**Discussion by kodepirate** [github/india#844] [Startups]
#11  ****Introduce Yourself****

Lets share and help  eachothers ideas and experiences about  startups 
How  was the experience?
How is it going ?
Why we should start a startups ?
What makes/motivates you to start a startup?
Why Enterpreneurship?

Share your knowledge and experiences anything you know

**Comment by FRTMentor** [comment]
Hi @kodepirate ,

Please share your testimonial in the below link.
https://futurereadytalent.in/Home/StudentProfile
